plugin_runner/plugin_runner.py

```diff
@@ -174,7 +180,7 @@ async def HandleEvent(
             )
 
             try:
-                handler = handler_class(event, secrets)
+                handler = handler_class(event, secrets, ENVIRONMENT)
```

Comment:
does changing the signature here break any existing plugins?

Reply:
No, because it changes the base init method, but plugins only implement the `compute` method. We make no guarantees if someone redefines init.